src/main/java/com/todolist/app/servlet/ToDoServlet.java

```diff
@@ -1,21 +1,50 @@
 package com.todolist.app.servlet;
+
+
+import com.todolist.app.service.ToDoStoreSingleton;
 import jakarta.servlet.*;
 import jakarta.servlet.http.*;
 import jakarta.servlet.annotation.*;
 import java.io.IOException;
 import java.io.PrintWriter;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import com.todolist.app.model.Item;
 
 
-
-
-
-@WebServlet(name = "ToDoServlet", value = "/hello")
+@WebServlet(name = "ToDoServlet", value = "/hello/")
```

Comment:
let's change it to `/notification` servlet so that we can use it in other operations like update for example.